backend/app/services/chat/trigger/emitter.py

```
@@ -424,6 +453,155 @@ async def _update_execution_status(
                 f"status to {status}: {e}"
             )
 
+    async def _send_notification(
+        self,
+        status: str,
+        result_summary: Optional[str] = None,
+        error_message: Optional[str] = None,
```

Ruff (0.14.14)

[warning] Unused method argument: `error_message`

(ARG002)